src/components/Card/Card.scss

```diff
@@ -84,6 +87,12 @@ $block: '.#{variables.$ns}card';
                 content: '';
             }
         }
+
+        @at-root &#{$block}_size_l {
```

Comment:
Let's also move `--_--box-shadow: 0px 1px 5px var(--g-color-sfx-shadow);` declaration from above under `@at-root &#{$block}_size_m` section here to be symmetrical with `&_type_container` styles below